src/ckeditor/smartpicker/InsertItem.js

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
Please rename the file to InsertItemCommand.js

(to keep the naming somewhat in sync with the signature plugin)